benchmark/tool/C/CMakeLists.txt

```diff
@@ -41,7 +41,8 @@ if(ANDROID_ABI)
     target_link_libraries(${PROJECT_NAME}
                           -Wl,--start-group -Wl,--whole-archive -lpaddle_capi_layers
                           -Wl,--no-whole-archive -lpaddle_capi_engine -Wl,--end-group
-                          ${THIRD_PARTY_LIBRARYS})
+                          ${THIRD_PARTY_LIBRARYS}
+                          log)
```

Comment:
What's the difference of `log` and `-llog` in the `target_link_libraries`. Since I see the `-llog` is used in code of icoder  http://icode.baidu.com/repos/baidu/paddle/mobile/blob/master:android/CMakeLists.txt

Reply:
I think should be `-llog`.

Reply:
@qingqing01 @hedaoyuan In fact, `log` is enough for Android, like [here](https://github.com/Xreki/PaddleNativeDemo/blob/master/android/optical_character_recognization/CMakeLists.txt#L43 ). I think `log` is already a `cmake library`.

Reply:
>  I think log is already a cmake library.

Yes, `log` is ok.